### DISCUSSION TITLE
What is the current bootstrap 5 version for understrap and where can is see bootstrap5 files?

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#2008] [Q&A]
Hey all,
According to [bootstrap 5 official page](https://getbootstrap.com/docs/versions/) the latest version is 5.2

What is the version understrap work with and where can i see bootstrap 5 files in understrap?
I only see bootstrap 4

**Comment by bacoords** [comment]
The understrap parent theme includes both BS4 and BS5 (the 5.2 update will be released shortly). You can check all of the build scripts in the [package.json](https://github.com/understrap/understrap/blob/main/package.json) file to see how to build various assets. 

You'll also need to set the theme to BS5 in the Customizer or by [overriding the theme mod manually](https://github.com/understrap/understrap-child/blob/main/functions.php#L70) if you want BS5 on your site. BS4 is still the default.

The understrap child theme only includes assets for BS5, but you can check the releases page for older versions that were based on BS4.
